pbsmetrics/metrics.go

```diff
@@ -256,4 +256,5 @@ type MetricsEngine interface {
 	RecordUserIDSet(userLabels UserLabels) // Function should verify bidder values
 	RecordStoredReqCacheResult(cacheResult CacheResult, inc int)
 	RecordStoredImpCacheResult(cacheResult CacheResult, inc int)
+	RecordPrebidCacheRequestTime(length time.Duration)
```

Comment:
The intent is to capture the duration of the HTTP call out to Prebid Cache for alerts and to compare against the Prebid Cache duration metrics. I don't personally think labels are warranted here, but please let me know if you feel differently.

Reply:
No, I don't think labels will be needed for this.

Reply:
Actually, we might want a success/failure flag, so we can see what the failure population looks like. For example, if we have significant population of cache requests that go 50ms+, but are ok because the bid requests all completed well under the TTL, yet have a bunch of quick cache requests that failed for some other reason, we could wrongly deduce that the failures are the fault of the 50ms+ population.

Reply:
Pushed.